tool/cmd/kitex/args/args.go

```diff
@@ -103,6 +103,8 @@ func (a *Arguments) buildFlags(version string) *flag.FlagSet {
 		"Combine services in root thrift file.")
 	f.BoolVar(&a.CopyIDL, "copy-idl", false,
 		"Copy each IDL file to the output path.")
+	f.BoolVar(&a.KeepResp, "keep-resp", false,
+		"When rpc error,keep resp.")
```

Comment:
When the server-side handler returns both err and resp, the resp return is retained for use in middleware where both err and resp can be used simultaneously. Note: At the RPC communication level, if the handler returns an err, the framework still only returns err to the client without resp